easybuild/easyconfigs/r/RAxML-NG/RAxML-NG-0.5.1b-intel-2018a.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = 'CMakeMake'
+
+name = 'RAxML-NG'
+version = '0.5.1b'
+
+homepage = 'https://github.com/amkozlov/raxml-ng'
+description = """RAxML-NG is a phylogenetic tree inference tool which uses maximum-likelihood (ML) optimality
+    criterion. Its search heuristic is based on iteratively performing a series of Subtree Pruning and
+    Regrafting (SPR) moves, which allows to quickly navigate to the best-known ML tree."""
+
+toolchain = {'name': 'intel', 'version': '2018a'}
+toolchainopts = {'usempi': True}
+
+source_urls = ['https://github.com/amkozlov/raxml-ng/releases/download/0.5.1']
```

Comment:
@ajaust We should try to avoid hardcoding the version in the source URL... The `b` bit makes that a bit annoying, but you can use this approach:

```python
source_urls = ['https://github.com/amkozlov/raxml-ng/releases/download/%s' % version[:-1]]
```

Reply:
You are right. I don't know why I had this still in there. 

Is there a preferred way to automatically insert the version number? I have fixed this in the easybuild file by using `v%(version)s`.
```
source_urls = ['https://github.com/amkozlov/raxml-ng/releases/download/v%(version)s']
```